modules/ROOT/pages/notifications-configuration-reference.adoc

```diff
@@ -0,0 +1,113 @@
+= Notifications Configuration Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, notifications, error messages, responses, exceptions, monitor
+
+This topic provides details on the elements to configure for notifications.
+//DOCS-4183 For more information on notifications, see xref:mule-server-notifications.adoc[Mule Server Notifications].
+
+== Notifications
+
+Registers listeners for notifications, and associates interfaces with specific events.
+
+=== Attributes of Notifications
+
+[%header%autowidth.spread]
+|===
+|Name |Description
+|`dynamic` |If the notification manager is dynamic, you can set up the listeners to be registered dynamically at runtime. Otherwise, some parts of Mule will cache the notification configurations for efficiency reasons and will not generate events for newly-enabled notifications or listeners. +
+*Type*: `boolean` +
+*Required*: no +
+*Default*: `false`
+|===
+
+=== Child Elements of Notifications
+Use the `notification` and `disable-notification` elements to configure the relationship between interfaces and events.
+[%header%autowidth.spread]
+|===
+|Name |Cardinality |Description
+|`notification` |0..* |Associates an event with an interface. Listeners that implement the interface receive instances of the event.
+|`disable-notification` |0..* |Blocks the association of an event with a particular interface. This filters events after the association is made with the interface, so it takes precedence over other elements, such as `notification-listener`.
+|`notification-listener` |0..* |Registers a bean as a listener with the notification system. Events are dispatched by reflection, which means that the listener receives all events associated with the interfaces it implements.
+|===
+
+== Notification
+
+Associates an event with an interface. Listeners that implement the interface will receive instances of the event.
+
+=== Attributes of Notification
+
+[%header%autowidth.spread]
+|===
+|Name |Description
+|`event-class` |The class name of the notification event to deliver to the interface. You can use this instead of the `event` attribute to specify a custom class. +
+*Type*: `class name` +
+*Required*: no +
+*Default*: none
+|`event` |The notification event to deliver. +
+*Type*: `notificationTypes` +
+*Required*: no +
+*Default*: none
+|`interface-class` |The class name of the interface that receives the notification event. +
+*Type*: `class name` +
+*Required*: no +
+*Default*: none
+|`interface` |The name of the interface that receives the notification event. +
+*Type*: `notificationTypes` +
+*Required*: no +
+*Default*: none
+|===
+
+There are no child elements for `notification`.
+
+== Disable Notification
+
+Blocks the association of an event with a particular interface. This filters events after the association is made with the interface, so it takes precedence over over other elements, such as `notification-listener`.
```

Comment:
Word "over: duplicated... _so it takes precedence over over other elements, such as `notification-listener`_

Reply:
OK